README.md

```diff
@@ -118,6 +118,66 @@ function ExampleWrapper({
 
 [Try it on Code Sandbox](https://codesandbox.io/s/x70ly749rq)
 
+## Advanced usage
+
+Some use-cases need the cached rows to be cleared for example a sorted list. Otherwise the InfiniteLoader won't call the function passed in `loadMoreItems` prop. This can be achieved by calling the `InfiniteLoader` component's method `resetloadMoreItemsCache`.
+
+```jsx
+function ExampleWrapper({
+  hasNextPage,
+  isNextPageLoading,
+  items,
+  loadNextPage,
+  sort,
+}) {
+  // We create a reference for the InfiniteLoader
+  const listRef = useRef(null);
+  
+  useEffect(() => {
+    if (listRef) {
+      // Each time the sort prop changed we called the method resetloadMoreItemsCache to clear the cache
+      listRef.current.resetloadMoreItemsCache()
+    }
+  }, [sort])
+  
+  const itemCount = hasNextPage ? items.length + 1 : items.length;
```

Comment:
This example shows a lot of things that aren't related to the `resetloadMoreItemsCache` method. Maybe we can just hide those behind a `// ...` comment and draw focus to the `useEffect` bit?

Reply:
Sure I'll do it 👍

Reply:
Does it feel better like this ?